packages/devextreme/js/ui/tab_panel.js

```diff
@@ -242,7 +251,19 @@ const TabPanel = MultiView.inherit({
         this.setAria('controls', id, $activeTab);
     },
 
-    _tabConfig: function() {
+    _getTabsIndicatorPosition() {
+        const { _tabsIndicatorPosition, tabsPosition } = this.option();
+
+        if(_tabsIndicatorPosition) {
+            return _tabsIndicatorPosition;
+        } else {
+            return TABS_INDICATOR_POSITION_BY_TABS_POSITION[tabsPosition];
```

Comment:
May be it's better to change as follows 
```
return _tabsIndicatorPosition ??TABS_INDICATOR_POSITION_BY_TABS_POSITION[tabsPosition];
```

Reply:
[9c9a6a123afafef3f4cb39c13550ef084dd60bf9]